devicetypes/smartthings/zwave-dimmer-switch-generic.src/README.md

```diff
@@ -5,6 +5,7 @@
 Works with: 
 
 * [Leviton Plug-in Lamp Dimmer Module (DZPD3-1LW)](http://www.leviton.com/OA_HTML/ProductDetail.jsp?partnumber=DZPD3-1LW)
+* [Leviton Universal Dimmer (DZMX1-LZ)](http://www.leviton.com/OA_HTML/ProductDetail.jsp?partnumber=DZMX1-1LZ)
```

Comment:
Should we use the WWST link? https://www.smartthings.com/works-with-smartthings/switches-and-dimmers/leviton-universal-dimmer